src/platforms/flutter/config.yml

```
@@ -1,3 +1,8 @@
 title: Flutter
-caseStyle: camelCase
+caseStyle: canonical
 supportLevel: production
+sdk: sentry.dart.flutter
+fallbackPlatform: dart
+categories:
+  - mobile
+  - browser
```

Not sure it's a valid category

```suggestion
  - browser
  - server
```

---

browser
      desktop
      mobile
      server
      serverless

are the valid ones